Buddies.Web/src/pages/postings.tsx

```diff
@@ -171,6 +170,7 @@ const PostingsTable = () => {
 
                               </Typography>
                               <Button
+                                disabled
```

Comment:
Just an FYI, if you're using a `Button` to just display elements that's not very good a11y. I would suggest changing the component.